docs/source/dev/workflow.rst

```diff
@@ -38,32 +38,86 @@ have a GitHub account and that you are able to create/push to repositories on Gi
 An easy wasy to approach the world of git actions and their integration with GitHub is to use
 `GitHub Desktop <https://desktop.github.com/>`_.
 
-Development of JADE relies heavily on branching; specifically, we use a branching model
-sometimes referred to as git flow. If you plan to contribute to JADE development,
-we highly recommend that you read this
+Branching model
+---------------
+.. figure:: /img/dev_guide/branching_model.svg
+    :width: 600
+    :align: center
+
+    Branching model showing Developing, Feature, Benchmark, Release and Hotfix branches
+
+Development of JADE relies heavily on branching; specifically, we use a modified 
+version of the branching model sometimes referred to as git flow. If you plan to 
+contribute to JADE development, we highly recommend that you read this
 `blog post <https://nvie.com/posts/a-successful-git-branching-model/>`_
-to get a sense of how the branching
-model works. There are two main branches that always exist: ``master`` and ``Developing``.
-The master branch is a stable branch that contains the latest release of the code.
-The develop branch is where any ongoing development takes place prior to a release and is
-not guaranteed to be stable. When the project leader decides that a release should occur,
-the ``Developing`` branch is merged into master.
-
-All new features, enhancements, and bug fixes should be developed on a branch that branches off
-of Developing. When the feature is completed, a pull request is initiated on GitHub that is
-then reviewed by an approver. If the pull request is satisfactory, it is then merged into develop.
-Note that a committer may not review their own pull request
-(i.e., an independent code review is required).
+to get a sense of how the branching model works. 
+
+There are two main branches that always exist: ``master`` and ``Developing``. 
+The ``master`` branch is a stable branch that contains the latest release of the 
+code. The ``Developing`` branch is where any ongoing development takes place 
+prior to a release and is not guaranteed to be stable. 
+
+Other branches will be created as required for different purposes which are 
+listed below. When any of these branches need to be merged (e.g., into the 
+``Developing`` branch), a pull request (PR) should be initiated on GitHub that is 
+then reviewed by an approver (see `Requirements for a successful merge`_). If the 
+pull request is satisfactory, it is then merged into develop. Note that a 
+committer may not review their own pull request (i.e., an independent code 
+review is required). 
+
+* **Feature** - All new features, enhancements, and bug fixes should be 
+  developed on a ``Feature`` branch off ``Developing``. Most branches on the 
+  repository will be of this type and consist of work by the core JADE team.
+* **Benchmark** - This type of branch is similar to that above, except it is 
```

Comment:
Perhaps be more explicit here that in terms of branching this is no different to adding a Feature to avoid any ambiguity.

Reply:
Reworded to be more explicit about similarities